corehq/apps/users/templates/users/enterprise_users.html

```diff
@@ -11,19 +11,24 @@
 
   <div class="panel panel-default ko-template" id="web-users-panel">
     <div class="panel-heading">
+      <h3 class="panel-title" style="padding-top: 7px;">{% trans 'Users' %}</h3>
+    </div>
+    <div class="panel-body">
       <div class="row">
-        <div class="col-sm-6">
-          <h3 class="panel-title" style="padding-top: 7px;">{% trans 'Users' %}</h3>
-        </div>
         <div class="col-sm-6">
           <search-box data-apply-bindings="false"
                       params="value: query,
                               action: function() { goToPage(1); },
                               placeholder: '{% trans_html_attr "Search Users..." %}'"></search-box>
         </div>
+        <div class="col-sm-6">
+            <button type="button" class="btn btn-default pull-right "
+                    data-bind="text: {% trans 'toggleDeactivatedText' %}, click: function() { showDeactivated(!showDeactivated()); }"
```

Comment:
This doesn't need to be translated, since it's a function name - but the user-facing text that's now inside the function does need to be translated.

Reply:
This is how I was looking at doing that. Does this make sense. `self.toggleDeactivatedText = ko.computed(function () {
            return self.showDeactivated() ? gettext("Hide Deactivated Mobile Workers") : gettext("Show Deactivated Mobile Workers");
        }, this);`

Reply:
yep, that looks good - one tweak is that I don't usually see the final `this` passed, it's probably passed in by default

Reply:
rather, to be more precise, `ko.computed` probably defaults to the current scope when one isn't passed in